testsuite/cluster-test/src/experiments/performance_benchmark_three_region_simulation.rs

```diff
@@ -97,7 +97,7 @@ impl Experiment for PerformanceBenchmarkThreeRegionSimulation {
     }
 
     fn deadline(&self) -> Duration {
-        Duration::from_secs(420)
+        Duration::from_secs(600)
```

Comment:
Have you tried this experiment with latest master, without increasing deadline?
We had a problem with network discovery that made startup very long, but it is resolved now. Do we still need to bump deadline to 10 minutes(that a lot...)

Reply:
I tired experiment with master but I reckon it was not the latest, I can re-do the experiment to confirm this point. Why I keep this change here is because I found we use 600s for other performance tests, so I assume 600s is a reasonable number for all performance tests. May I ask why we set 420s here which is much shorter than others at the first place?

Reply:
Ok, I see that other set to 10min, you are right. We can keep it here